### ISSUE TITLE
Unknown touch device id on mac with SDL

### ISSUE THREAD

**Issue by Pandawan** [dotnet/Silk.NET#442]
# Summary
Hovering my mouse over the rendering window causes a `Silk.NET.SDL.SdlException: Unknown touch device id #, cannot reset` error which crashes the app.

# Steps to reproduce
- Platform: Desktop, macOS Big Sur 11.2.3, Macbook Pro 2019 (with Touchbar) (no external devices connected)
- Framework Version: .NET 5.0.201
- Silk.NET Version: 2.2.0 (from NuGet)
- API: OpenGL with SDL
- API Version: Not sure... default for macos?

I created a simple project with `dotnet new`, copied the `Program.cs` content from [Tutorial 1.1](https://github.com/dotnet/Silk.NET/blob/main/examples/CSharp/Tutorial%201.1%20-%20Hello%20Window/Program.cs) and installed the latest NuGet Silk.NET package, added `SdlWindowing.Use();` at the top of the `Main` to use SDL—because GLFW currently has a breaking issue on mac/linux (which is going to get fixed in next release).

Anyway whenever I `dotnet run`, a window correctly opens; however two things happen:
1. It asks for OS "Input Monitoring" permissions
2. The app crashes the instant I move my cursor over the window while the app is focused

Here's the error message I get:
```
Unhandled exception. Silk.NET.SDL.SdlException: Unknown touch device id 2062041968, cannot reset
   at Silk.NET.Input.Sdl.SdlInputContext.Update(Double obj)
   at Silk.NET.Windowing.Internals.ViewImplementationBase.DoUpdate()
   at Silk.NET.Windowing.WindowExtensions.<>c__DisplayClass2_0.<Run>b__0()
   at Silk.NET.Windowing.Internals.ViewImplementationBase.Run(Action onFrame)
   at Silk.NET.Windowing.WindowExtensions.Run(IView view)
   at Engine.Program.Main(String[] args) in /Users/migueltenant/Dev/csharp/engine/Program.cs:line 31
```

# Comments
- It seems the touch device id changes every time I run it
- [Full code can be found here](https://pastebin.com/06YMKzWT)
- Allowing/Denying input monitoring permissions does not change anything
- I've done some digging and it may or may not have to do with [asking for an error when there isn't one? (see comments)](https://stackoverflow.com/questions/63983054/sdl-geterror-returning-an-undocumented-error-on-macos)

**__If you know how to fix this issue, please submit a pull request instead!__**

**Comment by Perksey** [comment]
This could be a case of us making much ado of nothing, similar to #439.

The fix here would be adjusting the input code to only call `ThrowError` if we detect a failure.

Contributions would be appreciated.

**Comment by Perksey** [comment]
We have removed the problematic ThrowError call and this should be fixed in 2.4. If when 2.4 comes out you find this isn't fixed, reopen this issue :D
